prow/config.yaml

```diff
@@ -257,6 +257,19 @@ tide:
     - do-not-merge/invalid-owners-file
     - do-not-merge/work-in-progress
     - needs-ok-to-test
+  - repos:
+    - kubernetes-incubator/service-catalog
+    labels:
+    - lgtm
+    - approved
+    - "cncf-cla: yes"
+    - LGTM1
+    - LGTM2
+    missingLabels:
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
```

Comment:
I did not add ` - do-not-merge/invalid-owners-file` because I do not know what it means.

Reply:
which plugin? Should we have it?

Reply:
`verify-owners`, uh probably? it's a new thing. I only saw it in passing, seems like a good idea but I haven't interacted with it yet. some details should be here https://prow.k8s.io/plugins

Reply:
makes sense, sorry. I'll turn that one on as well. prevent people from fearing changing those.